### PR TITLE
Urbanopt-example-project fixes.

### PR DESCRIPTION
**Pull request by tanushree04**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ if allow_local && File.exist?('../openstudio-model-articulation-gem')
   # gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'develop'
   gem 'openstudio-model-articulation', path: '../openstudio-model-articulation-gem'
 else
-  gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'develop'
+  gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'Edits'
```
Comment (macumber) [inline Gemfile]: @tanushree04 this is just for temporary debugging right https://github.com/NREL/openstudio-model-articulation-gem/commit/38003aec360cf26ba05e68f936806da3b8ed216c ?  If so, I will just delete that branch of the model articulation repo

File: Gemfile [hunk 2]
```diff
@@ -34,14 +34,15 @@ if allow_local && File.exist?('../urbanopt-scenario-gem')
...
 end
 
+
 if allow_local && File.exists?('../urbanopt-geojson-gem')
   # gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
-  gem 'urbanopt-geojson', path: '../urbanopt-geojson-gem'
+  gem 'urbanopt-geojson', path: '../urbanopt-scenario-gem'
 else
-  gem 'urbanopt-geojson', github: 'urbanopt/urbanopt-geojson-gem', branch: 'method-extractions'
+  gem 'urbanopt-geojson', github: 'urbanopt/urbanopt-geojson-gem', branch: 'example_project_fixes'
```
Comment (macumber) [inline Gemfile]: I am going to merge this https://github.com/urbanopt/urbanopt-geojson-gem/pull/21/files

File: Rakefile [hunk 3]
```diff
@@ -32,6 +32,7 @@ require 'openstudio/extension'
 require 'openstudio/extension/rake_task'
 require 'urbanopt/scenario'
 require 'urbanopt/geojson'
+require "json"
```
Comment (macumber) [inline Rakefile]: Is this needed here?

File: Gemfile (hunk 1 above)
Comment (tanushree04) [reply Gemfile]: There are also some previous commits for shortening file names .

**Comment by macumber** [comment]
Closing this and merging by hand.

**Pull request closed without merging**